code_generator/End2End/Tests/EntryPoint.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tests
```

Comment:
Again, if this is never invoked as an exe, might be cleaner to compile just to a `dll` as I believe you can run `nunit` on `dll`s

Then you can get rid of the empty file

Reply:
Yep, you can run nunit on DLLs just fine.